tools/update-rules-docs.js

```diff
@@ -38,9 +38,10 @@ async function formatTableRowFromFilename(ruleFilename) {
 	const ruleURL = new URL(ruleFilename, rulesDirURL);
 	const { default: rule } = await import(ruleURL);
 	const ruleName = path.parse(ruleFilename).name;
-	const { description, recommended } = rule.meta.docs;
+	const docs = rule?.meta?.docs;
 	const ruleLink = `[\`${ruleName}\`](./docs/rules/${ruleName}.md)`;
-	const recommendedText = recommended ? "yes" : "no";
+	const description = docs?.description || "_no description_";
+	const recommendedText = docs?.recommended ? "yes" : "no";
```

Comment:
We will ensure that all rules have `description`. Is there a reason you're validating whether it's present?

Reply:
Thanks! I noticed that when a rule has no `meta.docs` the build fails without indicating which rule is the culprit, so I added a fallback to avoid the error. But if all rules will have a `description` there is no need for a fallback. I've reverted that change now.